core/sql/common/ComMisc.cpp

```diff
@@ -228,3 +228,17 @@ NAString ComConvertTrafNameToNativeName(
   return convertedName;
 }
 
+NABoolean ComTrafReservedColName(
+     const NAString &colName)
```

Comment:
If column is not delimited, then it will be upcased when this method is called.
If column is delimited ("a"), then it will retain the original case.